docs/how-to/configure/store-limit.md

```diff
@@ -28,7 +28,7 @@ PD provides the following two methods to configure scheduling rate limits on sto
 
     > **Note:**
     >
-    > The modification immediately only takes effect on newly added stores, and will be applied to the entire cluster after TiKV restarts. If you want to apply speed limits on all stores or an individual store, consider using the second method below.
+    > The modification only takes effect on stores added after this configuration change, and will be applied to all stores in the  cluster after you restart TiKV. If you want this change to work  immediately on all stores or some individual stores before the change without restarting, combine this configuration with the following method. See [Sample usages](#sample-usages) for more details.
```

Comment:
Can we be more specific about which following method?

Reply:
Good point. updated. PTAL again.